tests/scripts/nightly/run_wh_b0_only.sh

```diff
@@ -12,6 +12,26 @@ echo "Running nightly tests for WH B0 only"
 env pytest tests/ttnn/integration_tests/unet                # -> failing: issue #7556
 SLOW_MATMULS=1 WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml env pytest tests/ttnn/integration_tests/stable_diffusion
 
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_basic_transformer_block.py
```

Comment:
Are there tests in this folder that cannot be run in nightly, ex. perf, device perf etc? Is this why you list so many?

Reply:
Yes, we need to check these tests in the Nightly, that's why I have added all the tests here.